### PR TITLE
Made stats gr8 again!

### PR DESCRIPTION
**Pull request by Bluscream**
- Added private message channels in ()
- Properly formatting streams URLs now
- Added status (only when not offline to avoid confusion)
- Added usercount with convinient new bot.users list

### CODE REVIEW

File: cogs/misc.py [hunk 1]
```diff
@@ -58,11 +58,22 @@ def __init__(self, bot):
...
             em.add_field(name=u'\u270F Keywords logged', value=str(self.bot.keyword_log))
             g = u'\U0001F3AE Game'
-            if '=' in game: g = '\ud83c\udfa5 Stream'
+            if '=' in game:
+                g = '\ud83c\udfa5 Stream'
+                game = game.split('=')
+                game = '[{}]({})'.format(game[0],game[1])
             em.add_field(name=g, value=game)
+            status = str(ctx.message.author.status)
+            if not status == 'offline':
```
Comment (chenzw95) [inline cogs/misc.py]: This looks rather arbitrary - do we know under what circumstances the status variable will give the "correct" value?

**Comment by Bluscream** [comment]
I still found some bugs in the status stuff, so i removed that code for now c:

File: cogs/misc.py [hunk 2]
```diff
@@ -58,10 +58,14 @@ def __init__(self, bot):
             em.add_field(name=u'\U0001F4E5 Msgs received', value=str(self.bot.message_count))
             em.add_field(name=u'\u2757 Mentions', value=str(self.bot.mention_count))
             em.add_field(name=u'\u2694 Servers', value=str(len(self.bot.guilds)))
-            em.add_field(name=u'\ud83d\udcd1 Channels', value=str(channel_count))
+            em.add_field(name=u'\ud83d\udcd1 Channels', value=str(channel_count)+' ('+str(len(self.bot.private_channels))+' private)')
```
Comment (chenzw95) [inline cogs/misc.py]: Is the private channel count really necessary? It might cause additional confusion to users. Mine, for example, is `88`.

**Comment by Bluscream** [comment]
Cause you have 88 DM channels open

**Pull request closed without merging**